### PR TITLE
release(hanoi-dot): Release manifest

### PR DESCRIPTION
**Pull request by bill-mahoney**
Hanoi-Dot release - Hold until Feb-17

Code Freeze - Feb - 10-17
Release - Feb 17.

Signed-off-by: Bill Mahoney <bill.mahoney@intel.com>

### CODE REVIEW

File: release/edgex-go.yaml [hunk 1]
```diff
@@ -1,9 +1,10 @@
 ---
 name: 'edgex-go'
-version: '1.3.0'
+version: '1.3.1'
 releaseName: 'hanoi'
-releaseStream: 'master'
+releaseStream: 'hanoi'
 repo: 'https://github.com/edgexfoundry/edgex-go.git'
+commitId: 'b46b7e8111e4043173f976c96fadf622ec12f3cd'
```
Comment (lenny-intel) [inline release/edgex-go.yaml:7]: Note that @tonyespy has a pending snaps PR on the `hanoi` branch waiting on a response to my question.
https://github.com/edgexfoundry/edgex-go/pull/3102

**Review by ernestojeda — APPROVED** [review]
LGTM

File: release/edgex-go.yaml (hunk 1 above)
Comment (lenny-intel) [reply release/edgex-go.yaml:7]: @bill-mahoney , with the merge of @tonyespy snap PR the commit for `hanoi` release is now `12a188d7528ee379d18cf3e1ac715e90dfef7258`:

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by lenny-intel — APPROVED** [review]
LGTM

**Pull request merged**